vsintegration/src/FSharp.Editor/DocComments/XMLDocumentation.fs

```diff
@@ -217,6 +218,23 @@ module internal XmlDocumentation =
                 EnsureHardLine collector
                 WriteElement collector el
 
+            match Seq.tryHead (doc.Descendants(XName.op_Implicit "remarks")) with
+            | None -> ()
+            | Some el ->
+                AppendHardLine collector
+                AppendOnNewLine collector (SR.RemarksHeader())
+                AppendHardLine collector
+                WriteElement collector el
```

Comment:
Neither the C# VS tooling nor, e.g., Ionide, shows "remarks" (although both give links to the documentation). While it does seem like seeing the "remarks" section inline could sometimes be useful, I wonder whether this change might start flooding these tooltips with giant walls of text, e.g.:

https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.getenumerator?view=net-8.0#remarks

If we did keep "remarks," it would probably be better to have it at the end (after "returns").

Reply:
Because remarks/examples can be so large Ionide puts them in the Info Panel instead - a detachable, standalone window that can be persistent. I agree that tool tips aren't necessarily the best place for that information here.

Reply:
> Neither the C# VS tooling nor, e.g., Ionide, shows "remarks" (although both give links to the documentation). While it does seem like seeing the "remarks" section inline could sometimes be useful, I wonder whether this change might start flooding these tooltips with giant walls of text, e.g.:
> 
> https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.getenumerator?view=net-8.0#remarks
> 
> If we did keep "remarks," it would probably be better to have it at the end (after "returns").

The C# VS tooling does show `remarks` part (but without the "remarks" heading), and it is before the `results` part...

Reply:
In C# it's configurable:
<img width="158" alt="image" src="https://github.com/dotnet/fsharp/assets/1760221/9f372ada-2c75-4b43-8562-78721160a817">

Reply:
...You're right—I even accidentally linked to it above 🙃.